testing/integration/upgrade_rollback_test.go

```diff
@@ -119,6 +120,12 @@ inputs:
 	}
 	require.NoError(t, err)
 
+	// ensure that upgrade details now show the state as UPG_ROLLBACK
+	state, err := startFixture.Client().State(ctx)
+	require.NoError(t, err)
+	require.NotNil(t, state.UpgradeDetails) // FIXME: there is no UpgradeDetails on State (yet!)
```

Comment:
The `UpgradeDetails` field on `state` will become available once https://github.com/elastic/elastic-agent/pull/3615 is merged.